src/HomeAssistant.cpp

```diff
@@ -67,45 +68,61 @@ FuFarmHomeAssistant::FuFarmHomeAssistant(Client &client) :
   // Setup sensors with the relevant information
   // Device classes are listed at https://www.home-assistant.io/integrations/sensor/#device-class
 #ifdef HAVE_WATER_LEVEL_STATE
-  // TODO: find a better device class for this
   waterLevel.setDeviceClass("moisture");
+  waterLevel.setIcon("mdi:cup-water");
+  waterLevel.setName("Water Tank Level");
```

Comment:
I'm really struggling to think of the best name for this. It is possible to have something that measures the water level (and we did use something using an ultrasonic distance sensor, which kind of works, but is tricky because turbulence in the water really messes things up), but this is a trigger that says whether the water has reached a particular point. This could be a low- or high-level indicator, to warn that a sump has either run dry or is overflowing. I'm not sure "Water Tank Level" explains that, but I'm struggling to think what might be better.

Reply:
The most ideal sensor would have been a contact one such as https://www.amazon.co.uk/KAOLALI-Water-Level-Sensors-Sensor/dp/B09TKYSS84 then do some single ADC (not sure if we'd need isolation). However, I am unsure if the length would be enough for the tank sizes in use. Maybe once it is past the max point (1m), we could just consider it as overflowing or we could stack 2-3 of them (assuming we have enough ADC pins) then add up the lengths.
Instead of `Water Tank Level`, for this case, how about `Samp Level Indicator`?